specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/examples/StorageAccountCreate.json

```diff
@@ -44,7 +46,7 @@
           "secondaryLocation": "centraluseuap",
           "statusOfPrimary": "available",
           "statusOfSecondary": "available",
-          "supportsHttpsTrafficOnly": false
+          "supportsHttpsTrafficOnly": true
```

Comment:
It was changed intentionally. Secure transfer required will be set as true by default.